src/components/asset-list/RecyclerAssetList.js

```diff
@@ -562,317 +472,101 @@ class RecyclerAssetList extends Component {
         if (
           openFamilyTabs[collectibles.data[i].familyName] === true &&
           !prevProps.openFamilyTabs[collectibles.data[i].familyName]
         ) {
-          let collectiblesHeight = 0;
-          for (let j = 0; j < i; j++) {
-            if (
-              openFamilyTabs[collectibles.data[j].familyName] &&
-              collectibles.data[j].tokens
-            ) {
-              collectiblesHeight +=
-                TokenFamilyHeader.height +
-                collectibles.data[j].tokens.length * UniqueTokenRow.height +
-                TokenFamilyWrapPaddingTop -
-                2;
-            } else {
-              collectiblesHeight += TokenFamilyHeader.height;
-            }
-          }
-          let investmentHeight = 0;
-          if (investments.data) {
-            for (let k = 0; k < investments.data.length; k++) {
-              if (!openInvestmentCards[investments.data[k].uniqueId]) {
-                investmentHeight +=
-                  UniswapInvestmentCard.height + InvestmentCard.margin.vertical;
-              } else {
-                investmentHeight +=
-                  InvestmentCardHeader.height + InvestmentCard.margin.vertical;
-              }
-            }
-          }
-          let balancesHeight = 0;
-          if (balances.data) {
-            balancesHeight += CoinRow.height * (balances.data.length - 1);
-            if (
-              balances.data[balances.data.length - 1].smallBalancesContainer ||
-              (balances.data[balances.data.length - 2] &&
-                balances.data[balances.data.length - 2].smallBalancesContainer)
-            ) {
-              balancesHeight += balances.data[balances.data.length - 1]
-                .smallBalancesContainer
-                ? ListFooter.height + 4
-                : 20;
-              if (openSmallBalances) {
-                balancesHeight +=
-                  CoinRow.height *
-                  balances.data[
-                    balances.data.length -
-                      (balances.data[balances.data.length - 1]
-                        .smallBalancesContainer
-                        ? 1
-                        : 2)
-                  ].assets.length;
-              }
+          const safeIndex = i;
+          const safeCollectibles = collectibles;
+          const familyIndex = findIndex(this.state.dataProvider._data, function(
+            data
+          ) {
+            return (
+              data.item?.familyName ===
+              safeCollectibles.data[safeIndex].familyName
+            );
+          });
+
+          const focusedFamilyItem = this.state.dataProvider._data[familyIndex]
+            .item;
+          const focusedFamilyHeight = ViewTypes.UNIQUE_TOKEN_ROW.calculateHeight(
+            {
+              amountOfRows: Math.ceil(
+                Number(focusedFamilyItem.childrenAmount) / 2
+              ),
+              isFirst: false,
+              isLast: false,
```

Comment:
the new calculateHeight function for UNIQUE_TOKEN_ROW is not doing anything with the params isLast or paddingBottom

Reply:
Sorry - removed